integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -261,3 +265,116 @@ def get_address_from_coordinates(request, region_slug):
             "country": address.get("country"),
         }
     )
+
+
+def show_poi_form_ajax(request, region_slug, language_slug, poi_title):
+    """Render a POI form template
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :param region_slug: The slug of the current region
+    :type region_slug: str
+
+    :param language_slug: The slug of the current language
+    :type language_slug: str
+
+    :param poi_title: The title of the POI that the template defaults to
+    :type poi_title: str
+
+    :return: The html template of a POI form
+    :rtype: ~django.http.HttpResponse
+    """
+    poi_form = POIForm()
+    poi_translation_form = POITranslationForm(data={"title": poi_title})
+    language = get_object_or_404(Language, slug=language_slug)
+    url_link = f"{settings.WEBAPP_URL}/{region_slug}/{language_slug}/{poi_translation_form.instance.url_infix}/{poi_title}"
+
+    media_context_mixin = MediaContextMixin()
+    media_context_mixin.request = request
+    media_config_data = media_context_mixin.get_context_data()
+
+    poi_context_mixin = POIContextMixin()
+    poi_context_mixin.request = request
```

Comment:
Hmm, I think it would make more sense to use a class based view to embed these contexts.
After all, there is no reason why a class based view with a `get()` and `post()` component should not be used as ajax view.
```
class POIFormAjaxView(TemplateView, POIContextMixin):
```